File: QuizApp.DataAccess/Entities/Answer.cs
namespace QuizApp.DataAccess.Entities
{
    public class Answer : BaseEntity
    {
        public string QuestionId { get; set; }
        public string AnswerText { get; set; }
        public bool IsCorrect { get; set; }

        public Question Question { get; set; }
    }
}

File: QuizApp.API/client-app/src/models/answer.ts
import {Answer} from "../appTypes";

export default class AnswerModel implements Answer {
    AnswerText: string;
    Id: string;
    IsCorrect: boolean;
    QuestionId: string;

    constructor(answerText: string,
                id: string,
                isCorrect: boolean,
                questionId: string) {
        this.Id = id;
        this.AnswerText = answerText;
        this.IsCorrect = isCorrect;
        this.QuestionId = questionId;
    }
}